program/src/instruction.rs

```diff
@@ -408,13 +429,15 @@ accounts_struct! {
         // Needs to be writable so we withdraw from it.
         pub validator_vote_account {
             is_signer: false,
+            // Is writable due to withdraw to reserve (vote_instruction::withdraw)
```

Comment:
There’s already a comment about it three lines above; let’s remove that comment then.

Reply:
fixed